pkg/ui/src/Header/Header.svelte

```diff
@@ -0,0 +1,54 @@
+<script lang="ts">
+	import Logo from './Logo.svelte';
+	export let title = 'Stock';
+	const links = [
+		{
+			label: 'Stock',
+			href: '/inventory/stock'
+		},
+		{
+			label: 'Inbound',
+			href: '/inventory/inbound'
+		},
+		{
+			label: 'Outbound',
+			href: '/inventory/outbound'
+		}
+	];
+	export let currentUrl = 'stock';
+</script>
+
+<div class="bg-gray-900 p-10 pt-5 ">
+	<div class="border-b border-sky-900 pb-5 flex sm:items-center ">
+		<Logo />
+
+		<div class="pl-4">
+			<nav class="flex space-x-4">
+				{#each links as { label, href }}
+					<a
+						{href}
+						class={[
+							'px-3 py-2 text-sm rounded-md',
+							currentUrl === label.toLowerCase()
+								? 'bg-teal-500 text-gray-900'
+								: 'text-white hover:text-teal-500'
+						].join(' ')}
+						aria-current={currentUrl === label.toLowerCase() ? 'page' : 'false'}
+					>
+						<span>{label}</span>
+					</a>
+				{/each}
+			</nav>
+		</div>
+	</div>
+	<div class="mt-2 md:flex md:items-center md:justify-between">
```

Comment:
You forgot the responsive classes here as well

Reply:
are you referring to making the title have a different hover style?

Reply:
No...`md:` prefixes